CONTRIBUTOR_ROLES.md

```diff
@@ -0,0 +1,224 @@
+
+### Contributor Responsibilities
+
+- Must follow the [Mozilla code of conduct](CODE_OF_CONDUCT.md)
+- Follow [contribution guidelines](CONTRIBUTING.md).
+
+### Contributor duties (one or more of the below)
+
+- File issues
+- Address issues through [pull requests](CONTRIBUTING.md#pull-request-lifecycle)
+- Contributing to documentation
+- Support other community members and contributors
+- Provide feedback on pull requests
+- Help promote the project
+- Help with community events
+
+### Contributor privileges
+
+- Invitation to contributor events.
+- Eligibility to become an Organization Member.
+
+## Organization Member
+
+An Organization Member is an established contributor who regularly participates in the project. Organization Members have privileges on the organization level and, as such, are expected to act in the interests of the whole project.
+
+### Requirements to become an Organization Member
+
+Meet one or more of the following requirements on a monthly basis:
+
+- Two or more accepted pull requests.
+- Reviewed three or more pull requests.
+- Resolved two or more issues.
+- Must have been contributing for at least two months.
+- Must be actively contributing to at least one project area.
+- Must have two sponsors who are also Organization Members.
+- Have two factor authentication enabled for their GitHub account.
+- Have enabled [signed commits](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits).
+
+#### Process for nominating an Organization Member
+
+- The nominator will open a pull request using the appropriate [template](https://github.com/mdn/mdn-community/roles/) against the [MDN Web Docs community repository](https://github.com/mdn/mdn-community/).
+- The nominee will add a comment to the pull request agreeing to all responsibilities of becoming an Organization Member.
+- To be accepted, the pull request needs three approvals. This can be any combination of Organization Members, Invited Experts, or Maintainers.
+- Once the pull request is approved, the new Organization Member is added to the [organization-members team](https://github.com/orgs/mdn/teams/organization-members).
+
+In addition to the responsibilities of a Contributor, Organization Members also have the following responsibilities:
+
+### Organization Member responsibilities
+
+- Continues to contribute regularly, as demonstrated by having at least 24 contributions a year, as indicated by the [insights](https://github.com/orgs/mdn/projects/15/insights) shown on the MDN Motherboard project board.
+
+### Organization Member privileges
+
+- May be assigned issues and reviews.
+- Will be added to the [organization-members team](https://github.com/orgs/mdn/teams/organization-members).
+- Can recommend other contributors to become Orginization Members.
```

Comment:
(Note from Friday morning session) - Reconsider this once we have set up Holopin.

```suggestion
```